README.md

```diff
@@ -76,6 +76,23 @@ steps:
             - /workdir/node_modules
 ```
 
+It also supports caching on specific JSON keys which can be specified following a `#` character using [jq syntax](https://jqlang.github.io/jq/manual/#object-identifier-index). This requires [jq](https://jqlang.github.io/jq/) to be installed on the build agent. You cannot use this in conjunction with Bash globbing.
+
+```yaml
+steps:
+  - command: npm test
+    plugins:
+      - seek-oss/docker-ecr-cache#v2.2.0:
+          cache-on:
+            - package.json#.dependencies
+            - package.json#.devDependencies
+            - package.json#.pnpm.overrides
+            - yarn.lock
```

Comment:
Would it be useful to list our recommended set of cache keys for a typical pnpm project here?

```suggestion
  - command: pnpm install --offline && pnpm test
    plugins:
      - seek-oss/docker-ecr-cache#v2.2.0:
          cache-on:
            - .npmrc
            - package.json#.dependencies
            - package.json#.devDependencies
            - package.json#.packageManager
            - package.json#.pnpm.overrides
            - pnpm-lock.yaml
```